backend/src/test/java/ca/bc/gov/hlth/mohums/controller/MetricsControllerTest.java

```diff
@@ -0,0 +1,60 @@
+package ca.bc.gov.hlth.mohums.controller;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInstance;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.jupiter.api.TestInstance.Lifecycle;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.test.autoconfigure.web.reactive.AutoConfigureWebTestClient;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.dao.DataAccessException;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.test.context.junit.jupiter.SpringExtension;
+
+import java.sql.DriverManager;
+import java.sql.SQLException;
+
+@ExtendWith(SpringExtension.class)
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+@AutoConfigureWebTestClient
+@TestInstance(Lifecycle.PER_CLASS)
+class MetricsControllerTest {
```

Comment:
These tests are good, but what I actually meant was to add more tests to `MoHUmsIntegrationTests` that test the `MetricsController` using the test's HTTP client.

Reasons:
* Tests the controller configuration
* Tests authorization to a small degree
* Tests the SQL queries

This said, while these tests are good, they will become redundant if you do the above. (The tests in this class are just testing the connection, but the tests I'm describing above cover that too.)